index.ts

```diff
@@ -462,6 +502,14 @@ function tryParseInt(value: string): number {
   return parsedValue;
 }
 
+function tryParseBool(value: string): boolean {
+  const validValues = ["true", "false"];
```

Comment:
just in case it's interesting, I think you can also do `typeof JSON.parse("false") === "boolean" `

Reply:
interesting!